tests/style/dataset/test_reader.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+from style.constants import FILE_PATH_MOCK_DB
+from style.dataset.reader import DatasetReader
+
+
+@pytest.fixture
+def dataset():
+    return DatasetReader.load_files(FILE_PATH_MOCK_DB)
```

Comment:
not a big fan of this variable name. it is not a DB.

Reply:
I changed with "DS"